capsules/src/lsm303agr.rs

```diff
@@ -436,7 +491,7 @@ impl i2c::I2CClient for Lsm303agrI2C<'_> {
                 self.i2c_magnetometer.disable();
                 self.state.set(State::Idle);
                 if self.config_in_progress.get() {
-                    self.set_range(self.mag_range.get());
+                    let _ = self.set_range(self.mag_range.get());
```

Comment:
Same as above. Shouldn't an error result in an upcall with an error?

Reply:
I updated it.